### PR TITLE
update ember-cli et al

### PR DESCRIPTION
**Pull request by marcoow**
🆙

### CODE REVIEW

File: node-tests/blueprints/authenticator-test.js [hunk 1]
```diff
@@ -65,19 +66,21 @@ export default Base.extend({' + EOL + '\
     }));
   });
 
-  it('throws when the authenticator is specified as its own base class', function() {
-    return expect(emberNew().then(() => {
-      return generateAndDestroy(['authenticator', 'application', '--base-class=application'])
-    })).to.eventually.be.rejectedWith(
-      'Authenticators cannot extend from themself. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".'
-    );
+  xit('throws when the authenticator is specified as its own base class', function() {
```
Comment (marcoow) [inline node-tests/blueprints/authenticator-test.js]: These tests currently fail due to ember-cli/ember-cli#7019

File: blueprints/authenticator/index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ module.exports = {
         body: EOL + '  restore(data) {' + EOL + '  },' + EOL + EOL + '  authenticate(/*args*/) {' + EOL + '  },' + EOL + EOL + '  invalidate(data) {' + EOL + '  }'
       };
     } else if (name === baseClass) {
-      throw new SilentError('Authenticators cannot extend from themself. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".');
+      throw new SilentError('Authenticators cannot extend from themselves. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".');
```
Comment (Turbo87) [inline blueprints/authenticator/index.js]: seems unrelated to the PR purpose of updating depenendencies?

File: node-tests/blueprints/authenticator-test.js [hunk 3]
```diff
@@ -65,19 +66,21 @@ export default Base.extend({' + EOL + '\
...
+    return emberNew().then(() => {
+      expect(generateAndDestroy(['authenticator', 'application', '--base-class=application'])).to.be.rejectedWith(
+        SilentError,
+        'Authenticators cannot extend from themselves. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".'
+      );
+    });
```
Comment (Turbo87) [inline node-tests/blueprints/authenticator-test.js]: the changed exception also seems unrelated to the PR purpose

File: package.json [hunk 4]
```diff
@@ -38,53 +38,54 @@
...
-    "ember-cli-content-security-policy": "~0.6.0",
-    "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.0.0",
-    "ember-cli-fastboot": "1.0.0-rc.5",
-    "ember-cli-htmlbars": "^2.0.0",
+    "chai": "^4.1.0",
+    "cors": "^2.8.4",
+    "ember-cli": "~2.14.0",
+    "ember-cli-addon-tests": "^0.7.0",
+    "ember-bootstrap": "^1.0.0-beta.2",
```
Comment (Turbo87) [inline package.json]: wrong sort order

File: package.json [hunk 5]
```diff
@@ -38,53 +38,54 @@
...
-    "ember-cli-chai": "^0.4.0",
-    "ember-cli-content-security-policy": "~0.6.0",
-    "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.0.0",
-    "ember-cli-fastboot": "1.0.0-rc.5",
-    "ember-cli-htmlbars": "^2.0.0",
+    "chai": "^4.1.0",
+    "cors": "^2.8.4",
+    "ember-cli": "~2.14.0",
+    "ember-cli-addon-tests": "^0.7.0",
```
Comment (Turbo87) [inline package.json]: this looks like a new dependency, not an updated one? 🤔

File: node-tests/blueprints/authenticator-test.js (hunk 3 above)
Comment (marcoow) [inline node-tests/blueprints/authenticator-test.js]: Yeah, was trying to fix the tests until I figured out they were breaking because of ember-cli

File: blueprints/authenticator/index.js (hunk 2 above)
Comment (marcoow) [inline blueprints/authenticator/index.js]: yeah, reverted

File: package.json (hunk 5 above)
Comment (marcoow) [inline package.json]: fixed

File: package.json (hunk 4 above)
Comment (marcoow) [inline package.json]: fixed

File: node-tests/blueprints/authenticator-test.js (hunk 3 above)
Comment (marcoow) [inline node-tests/blueprints/authenticator-test.js]: reverted

**Comment by marcoow** [comment]
@Turbo87: good to approve?
